resources/assets/less/bem/login-box.less

```diff
@@ -47,6 +47,12 @@
       width: 300px;
       margin-top: 5px; // sync with Nav2#centerPopup
     }
+
+    &--captcha {
+      @media @desktop {
+        width: 340px;
```

Comment:
this is 4 pixels off (the width is 304 but 340px only provides 300px wide content)

Reply:
Looks like 304px is to account for the drop shadow on their light-mode theme, setting the box to 344px makes the input boxes above the recaptcha box too wide... so I guess 342px to match the visual widths?